core/templates/dev/head/pages/query/Query.js

```diff
@@ -0,0 +1,88 @@
+// Copyright 2016 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Data and controllers for the Oppia admin page.
+ */
+
+oppia.controller('Query', [
+  '$scope', '$http', function($scope, $http) {
+    _QUERY_DATA_URL = '/querydatahandler';
```

Comment:
Missing var. Also, we have stopped putting underscores in front of private functions, here and below. Instead, functions private to the controller are declared using var, and functions that need to be accessible from the template are declared on $scope.